packages/maskbook/src/database/Persona/Persona.db.ts

```diff
@@ -125,7 +126,7 @@ export async function consistentPersonaDBWriteAccess(
             console.warn('The transaction ends too early! There MUST be a bug in the program!')
             console.trace()
             // start a new transaction to check consistency
-            t = createTransaction(await db(), 'readwrite')('profiles', 'personas')
```

Comment:
call `await db()` twice would cause error:
`DOMException:ConstraintError: Key already exists in the object store.(-1)`

Reply:
whoo, that's strange. I thought it was side-effect-free when the DB keeps open